src/ucp/wireup/select.c

```diff
@@ -1552,7 +1566,7 @@ ucp_wireup_get_current_num_lanes(ucp_ep_h ep, ucp_lane_type_t type)
      */
     if ((ep->cfg_index == UCP_WORKER_CFG_INDEX_NULL) ||
         ucp_ep_has_cm_lane(ep)) {
-        return UCP_PROTO_MAX_LANES;
+        return ucp_wireup_max_lanes(select_params, type);
```

Comment:
looks a bit error prone that it can return different values depending on lane type  (i.e. 5 for fast lanes)

Reply:
Fixed